From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Fri, 18 Apr 2014 23:45:16 +0100
Subject: configure.ac updates

Check that autoconf is running in the right directory, for safety.
Look for macros in m4/.
Flush the deprecated AM_MAINTAINER_MODE.
---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0bb909e..30076c6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,7 +1,8 @@
 AC_INIT([scrypt],[1.1.6])
+AC_CONFIG_SRCDIR([lib/scryptenc/scryptenc.c])
 AC_CONFIG_AUX_DIR([config.aux])
+AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([foreign subdir-objects -Wall])
-AM_MAINTAINER_MODE
 AC_PROG_CC
 
 # Check for clock_gettime.  On some systems, this is provided via librt.
